hoverxref/_static/js/hoverxref.js_t

```diff
@@ -186,23 +176,23 @@ $(document).ready(function() {
     {% endif %}
 
     function showModal(element) {
-        var project = element.data('project');
-        var version = element.data('version');
-        var doc = element.data('doc');
-        var docpath = element.data('docpath');
-        var section = element.data('section');
-        var url = element.data('url');
-
-        var url = getEmbedURL(project, version, doc, docpath, section, url);
+        // TODO: we are failing on some elements to set the URL from the
+        // backend, but we can use `href` instead.
+        var url = element.data('url') || element.prop('href');
+        var url = getEmbedURL(url);
         $.get(url, function(data) {
             var content = $('<div></div>');
-            content.html(data['content'][0]);
+            content.html(data['content']);
 
             var h1 = $('h1:first', content);
             var title = h1.text()
             if (title) {
                 var link = $('a', h1).attr('href') || '#';
-                var a = $('<a></a>').attr('href', link).text('{{ hoverxref_modal_prefix_title }}' + title.replace('¶', ''));
+
+                // Remove permalink icon from the title
+                var title = title.replace('¶', '').replace('', '');
```

Comment:
This is a hack to avoid showing a weird char next to the title. We may want to make this configurable in the future if we realize that other themes use different chars for this. However, I don't think it's a problem currently nor is worth the effort to make it better at this moment.

Reply:
I wonder if we can just do a regex that checks for a single non-standard char at the end of the string, or something.

Reply:
I think this is because you are transforming the title from html to text. The backend could remove those (we do this with search) or the client could just inject the html as is.

Reply:
I opened https://github.com/readthedocs/sphinx-hoverxref/issues/148 to not forget about this as I did this time 😄

Reply:
@ericholscher 

> I wonder if we can just do a regex that checks for a single non-standard char at the end of the string, or something.

I think this will cover most of the cases, yes.

@stsewd 

> I think this is because you are transforming the title from html to text. The backend could remove those (we do this with search) or the client could just inject the html as is.

I don't think we want to do this in the backend. I suppose that these chars could be useful in some contexts (they are the characters for permalinks); so I'd prefer to keep them in the response.

Also, note the content is injected in the tooltip as HTML to be rendered properly.